data/help.txt

```diff
@@ -125,3 +125,11 @@ help "trading"
 
 help "friendly disabled"
 	`This ship is disabled and needs your help to patch them up! Pressing <Board selected ship> will fly you to their ship and get them up and running again. Just be sure to do it when no one is trying to kill you, or they could get caught in the crossfire!`
+
+help "fighter fleet logistics"
+	`Fighter fleet logistics has been enabled.  This enables carriers and fighters to support your escorted fleet.  The following logistics are supported.`
+	``
+	`	Fleet refueling if your fighters have fuel and ramscoop.`
+
+help "try out fighter fleet logistics"
+	`You have a carrier which can support your fleet through logistics.  In Preferences and Settings under AI section, turn on "Figher fleet logistics" to learn more.`
```

Comment:
It's too expensive to split.  This PR has extremely heavy play testing by this point.  The tutorials are part of the AI overhaul which this PR became.

Reply:
It isn't.

Reply:
To elaborate, you've added and added and added to this PR, but by doing so you conditionalize accepting functionality which is perhaps desirable with functionality that may not fit with the game design, or which are larger changes to the mechanics and shouldn't be bundled.

Bundling changes together because getting your change proposals reviewed takes longer than you want isn't how you increase the speed at which your work gets reviewed, because every time your PR gets opened it has grown and changed and has to be reevaluated in full all over.

Reply:
This PR was opened in August.  The changes were small then; why wasn't it addressed sooner?

These changes you're asking to split out are tightly coupled with the performance optimizations only available in this pull request.  It's not possible to split them out.

Reply:
> This PR was opened in August.  The changes were small then; why wasn't it addressed sooner?

I think you meant April, and it [was mentioned to you then](https://github.com/endless-sky/endless-sky/pull/6726#pullrequestreview-972069930), that you were combining unrelated changes. You agreed then that you could have split it out but didn't because your code wasn't getting reviewed and merged fast enough.

> These changes you're asking to split out are tightly coupled with the performance optimizations only available in this pull request.  It's not possible to split them out.

Why are the performance optimizations _only in this PR_? I've stated in other issues/PRs (and Discord even) that I believe a three-tiered AI model is appropriate, with certain information and decision-making managed at the body level, some at the fleet level, and others at the government level. That demarcation of responsibilities would likely alleviate much of the issues you run into here, where you added lots of per-frame reads for niche behaviors.

You can certainly implement bits of this PR separately, and without performance hits even. It's important to remember that simplicity is paramount: making a decision complex compounds all aspects of interacting with the decision: extra memory reads, extra code, extra bugs, new edge cases, and extra testing requirements. It's always possible for us to write more code and create more complex criteria for behaviors. The hard question to ask is whether the complexity is warranted, or if a simpler rubric can be used to achieve similar results without the same performance and maintenance costs.

Reply:
I don't have the cycles to split this up near term so it will remain in limbo I guess until my other PRs get addressed.  Then, with maintaining less this can be revisited.  Not sure I fully understood your performance comments so I'll follow up with questions later.

Reply:
I think tutorials being added at the same time as the mechanics seems appropriate to me.